### DISCUSSION TITLE
Big announcements

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#37] [Announcements]
Hello everyone!
I have a few big announcements for you today, so listen closely.

# 1. Polaris is rebranding
In light of #36, Polaris will be rebranding to *-drumroll please-* Solaris!
The name is very similar to the original name, but unique enough to be brandable. Our new colors will be a mix of orange and red.

# 2. Ownership of the Solaris project is transferring
J Technologies is currently dissolving and being pieced back together into @NovaoTech. The repositories will be transferred on April 8th, 2022. On that day, the new logo will be unveiled as well.

# 3. A new code of conduct
The Solaris project will adopt @NovaoTech's soon-to-be-released code of conduct. More on that at a later date.

# What this means going forward.
For one, Solaris' pre-releases will be released at solaris.novao.xyz. The releases will automatically update from the master branch weekly.

Secondly, the Solaris project will consist of three separate repositories. The first, @NovaoTech/solaris, will be the planning and discussion point for Solaris. A town hall, per se. The second, @NovaoTech/dynamo, will contain the code for Solaris' backend, written with Python/flask and nicknamed dynamo. The final repository, @NovaoTech/corona, will contain the code for Solaris' frontend, written with SvelteKit and nicknamed Corona.

Finally, this means that development is finally going to pick up. From the 24th of April onwards, I will be dedicating my time to this project. We should hopefully have a beta release out by mid-June.

We *will* need some help to reach this goal, but I am confident that if everyone pitches in, we can create a wonderful new environment for open development and learning. Who knows, maybe one day we can expand to accept other languages?

Until next time,
Judah

**Comment by spazestudios** [comment]
🙃 this is all great!

**Comment by MikeDev101** [comment]
:+1:

**Comment by KSKgames** [comment]
Nice

**Comment by melis-the-menace** [comment]
Could I help design?

**Comment by micahlt** [comment]
Sounds super fun!  Are you willing to look at a somewhat dramatic change to the NovaoTech main site?  I have a few ideas.

**Reply by micahlt** [reply]
Okay with no fast response I'll go ahead and show what I've got:

![image](https://user-images.githubusercontent.com/28816297/165177733-d84b897c-c65b-42d2-a15a-ab0adcfefb94.png)
![image](https://user-images.githubusercontent.com/28816297/165177768-3f52a8ac-c943-44db-9709-66c5bd638132.png)

**Reply by icarusinthewings** [reply]
I like your design concepts! One thing I kind of dislike is the italics. It doesn't really fit with the personality of the brand I was going for.

**Reply by micahlt** [reply]
That's easily fixable 😅 yeah I wasn't sure if you wanted to do anything italic or not.  I like the current site, I just feel like it could use a bit more polish.

**Reply by icarusinthewings** [reply]
Yeah, I created that one in a rush. I'll be pushing some minor updates out in the coming weeks since my school isn't very busy now.
